src/ThgHostingTest.php

```diff
@@ -0,0 +1,118 @@
+<?php declare(strict_types=1);
+/**
+ * (c) The Hut Group 2001-2019, All Rights Reserved.
+ *
+ * This source code is the property of The Hut Group, registered address:
+ *
+ * 5th Floor, Voyager House, Chicago Avenue, Manchester Airport,
+ * Manchester, England, M90 3DQ
+ * @author Michal Dzierzbicki <michal.dzierzbicki@thg.com>
+ */
+namespace ThgHosting;
+
+class ThgHostingTest extends ThgHostingClient
+{
+    public const PROD_ENV  = 'prod';
+    public const DEV_ENV   = 'dev';
+    public const LOCAL_ENV = 'local';
+
+    public function __construct(string $env = self::DEV_ENV)
+    {
+        $path = '../test/keys/env.php';
```

Comment:
I will implement it but this is proper path with constants (because we want to go up by one level):
```suggestion
        $path = dirname(__DIR__) . DIRECTORY_SEPARATOR . 'test' . DIRECTORY_SEPARATOR
        . 'keys' . DIRECTORY_SEPARATOR . 'env.php';
```

Reply:
It was more about DIRECTORY_SEPARATOR than changing the actual path. I had to change it for myself becauses I used composer require and got the project in vendor.

Reply:
And if you were wondering why its better to use directory_separator then linux uses / and windows \